common/src/main/scala/com/aliyun/emr/rss/common/protocol/message/ControlMessages.scala

```diff
@@ -317,7 +317,28 @@ object ControlMessages extends Logging {
 
   case class ThreadDumpResponse(threadDump: String) extends Message
 
-  case class UserIdentifier(tenantId: String, name: String) extends Message
+  case class UserIdentifier(tenantId: String, name: String) extends Message {
+    assert(tenantId != null && !tenantId.isEmpty)
+    assert(name != null && !name.isEmpty)
+
+    override def toString: String = {
+      s"`$tenantId`.`$name`"
+    }
+  }
+
+  object UserIdentifier {
+    val USER_IDENTIFIER = "^\\`(.+)\\`\\.\\`(.+)\\`$".r
+
+    def apply(userIdentifier: String): Option[UserIdentifier] = {
+      if (USER_IDENTIFIER.findPrefixOf(userIdentifier).isDefined) {
+        val USER_IDENTIFIER(tenantId, name) = userIdentifier
+        Some(UserIdentifier(tenantId, name))
+      } else {
+        logError(s"Failed to parse user identifieer: `$userIdentifier`")
```

Comment:
Hers is a misspelling.

Reply:
> Hers is a misspelling.

Yea, I know. But can first refer to https://github.com/alibaba/RemoteShuffleService/pull/695 since this pr need change to yaml and other pr need this too,